eth_crowdfund_be/eth_crowdfund_api/src/views/CampaignView.py

```diff
@@ -0,0 +1,74 @@
+from flask import request, json, Response, Blueprint, g, jsonify
+from ..models.Campaign import Campaign, CampaignSchema
+
+
+
+campaign_api = Blueprint('campaigns_api', __name__)
+campaign_schema = CampaignSchema()
+
+@campaign_api.route('/', methods=['POST'])
+def create():
+  req_data = request.get_json()
+  data = campaign_schema.load(req_data)
+  # data, error = campaign_schema.load(req_data)
+  # BUT WITH ERROR: the above code threw an error, currently ignoring errors
+  error = None
+  if error: 
+    return custom_response(error, 400)
+
+  campaign = Campaign(data)
+  campaign.save() 
+  campaign_data = campaign_schema.dump(campaign)
+  return custom_response(campaign_data, 201)
```

Comment:
had to modify these 4 lines. adding `.data` to the end is no longer a usable syntax